$NetBSD: patch-bf,v 1.1 2006/01/18 23:24:12 rillig Exp $

Solaris does not have <stdint.h>, and the code does not seem to need it.

--- src/libcompat/regions.c.orig	2003-09-11 20:53:38.000000000 +0200
+++ src/libcompat/regions.c	2006-01-16 15:00:23.211191300 +0100
@@ -42,7 +42,6 @@
 #include <assert.h>
 #include <stdio.h>
 #include <string.h>
-#include <stdint.h>
 
 #define RPAGESIZE (1 << RPAGELOG)
 #define K 2
